test/tools/index.js

```diff
@@ -91,6 +91,9 @@ const main = () => {
     aeSubConfig = Buffer.concat([Buffer.from(["AE".length]), Buffer.from("AE"), Buffer.from([18])])
     var aeConfig = createCurrencyConfig("AE", "Ethereum", aeSubConfig, ledgerTestPrivateKey);
 
+    shibSubConfig = Buffer.concat([Buffer.from(["SHIB".length]), Buffer.from("SHIB"), Buffer.from([20])])
+    var shibConfig = createCurrencyConfig("SHIB", "Ethereum", shibSubConfig, ledgerTestPrivateKey);
```

Comment:
`var` aaaargh :P

Could replace _all_ `var` with `const` and banish `var` from the codebase.